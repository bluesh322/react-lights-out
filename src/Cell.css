.Cell {
  height: 100px;
  width: 100px;
  background-color: darkgray;
  cursor: pointer;
}

.Cell-lit {
  background-color: lightblue;
}

table th td {
  border-color: black;
}