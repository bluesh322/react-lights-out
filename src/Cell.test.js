import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Cell from "./Cell";

it("smoke test Carousel component", () => {
  render(<Cell/>);
});

it("snapshot test for Carousel component", () => {
    const {asFragment} = render(<Cell />);
    expect(asFragment()).toMatchSnapshot();
})