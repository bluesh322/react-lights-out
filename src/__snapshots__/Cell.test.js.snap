// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for Carousel component 1`] = `
<DocumentFragment>
  <td
    class="Cell "
    role="button"
  />
</DocumentFragment>
`;
